extensions/schema-compare/src/test/models/schemaCompareOptionsModel.test.ts

```diff
@@ -11,23 +11,35 @@ describe('Schema Compare Options Model', () => {
 	it('Should create model and set options successfully', function (): void {
 		const model = new SchemaCompareOptionsModel(testUtils.getDeploymentOptions());
 		should.notEqual(model.getOptionsData(), undefined, 'Options shouldn\'t be undefined');
-		should.notEqual(model.getObjectsData(), undefined, 'Objects shouldn\'t be undefined');
+		should.notEqual(model.getIncludeObjectTypesOptionsData(), undefined, 'Objects shouldn\'t be undefined');
```

Comment:
`Include objects shouldn't...`

But also should you be checking exclude objects as well?